pom.xml

```diff
@@ -552,6 +552,7 @@ under the License.
               -->
               <exclude>src/main/appended-resources/licenses/CDDL-1.0.txt</exclude>
               <exclude>src/main/appended-resources/licenses/EPL-1.0.txt</exclude>
+              <excludes>Jenkinsfile.its</excludes>
```

Comment:
Why not add proper license to new Jenkinsfile and drop this exclusion?
BTW comment in line 551 suggests it is license file, doesn't it?

Reply:
arfff really good catch :)